Support multiple breadcrumbs (#1)
naotoomura/breadcrumbs_on_rails: README.md

end
```

> ### Multiple breadcrumns

Reviewer: ```suggestion
### Multiple breadcrumbs
```

> For using multiple breadcrumbs, call `next_breadcrumbs` to create new breadcrumb stack.

Reviewer: ```suggestion
For using multiple breadcrumbs, call `next_breadcrumbs` to create a new breadcrumb stack.
```

```ruby
class MyController

  def index
    # ...

    add_breadcrumb "home", :root_path
    add_breadcrumb "my", :my_path
    add_breadcrumb "index", index_path

    next_breadcrumbs

    add_breadcrumb "home", :root_path
    add_breadcrumb "another", :another_path
    add_breadcrumb "index", index_path

  end

end
```

> When after create new breadcrumb stack, if you want to push to an existing stack.

Reviewer: ```suggestion
You can push a new element to an arbitrary stack by specifying an `index` option.
```

とかでしょうか？

### Restricting breadcrumb scope

> The `add_breadcrumb` method understands all options you are used to pass to a Rails controller filter. In fact, behind the scenes this method uses a `before_filter` to store the tab in the `@breadcrumbs_list` variable.

Reviewer: ここに `@breadcrumbs_list` と明記されているということは、`@breadcrumbs` はパブリックに使われてる可能性もあるかもしれませんね……そうだった場合、`@breadcrumbs_list` から `@breadcrumbs` は受け入れてもらえない可能性がありそうですね……。

後方互換性を残すために、`@breadcrumbs` には `@breadcrumbs_list.last` を常にもたせとく、などの対応が必要かもしれません。

> The `add_breadcrumb` method understands all options you are used to pass to a Rails controller filter. In fact, behind the scenes this method uses a `before_filter` to store the tab in the `@breadcrumbs` and `@breadcrumbs_list` variable.

Reviewer: ```suggestion
The `add_breadcrumb` method understands all options you are used to pass to a Rails controller filter. In fact, behind the scenes this method uses a `before_filter` to store the tab in the `@breadcrumbs` and `@breadcrumbs_list` variables.
```
